package-lock.json

```diff
@@ -18,7 +18,7 @@
 				"@graphql-tools/url-loader": "^7.17.18",
 				"@kiva/kv-components": "^3.109.3",
 				"@kiva/kv-shop": "^1.12.65",
-				"@kiva/kv-tokens": "^2.15.0",
+				"@kiva/kv-tokens": "^2.16.0",
```

Comment:
I set this to 2.16 instead of 2.15 b/c every reference to this package used 2.16... I could change it back to clean this up a bit more...